### PR TITLE
 Use m instead of cm 

### LINKED ISSUE #7: Use m instead of cm
It makes more sense to use meter vs cm and display the distance with 2 decimal places. Examples 0.45m, 50.2m.

### PR DESCRIPTION
**Pull request by tin0312**
fixes #7

### CODE REVIEW

**Comment by tin0312** [comment]
fixes #7

File: utils.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Reusable logic
```
Comment (DukeManh) [inline utils.js]: Suggestion: remove this comment

File: utils.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+function pixelToMeter(px) {
+        const DPI = 96; // Assumed dpi value for desktops, there is no way to correctly determine screen DPI
+        const INCH_PER_METER = 39.37; // 1 inch = 39.37 cm
+        const lengthInInch = px / DPI;
+        const lengthInMeter = lengthInInch / INCH_PER_METER; 
+        return lengthInMeter;
+    }
```
Comment (DukeManh) [inline utils.js]: Please format the file manually, we will setup linter later.

File: feed.html
Comment (DukeManh) [inline feed.html:1]: U can remove this file from the commit

**Review by DukeManh — CHANGES_REQUESTED** [review]

**Comment by DukeManh** [comment]
Can you squash the commits to one as well?

**Comment by tin0312** [comment]
fixed #7

**Review by DukeManh — APPROVED** [review]

**Pull request merged**